db/migrate/20240404142248_create_aliquots.rb

```diff
@@ -0,0 +1,19 @@
+class CreateAliquots < ActiveRecord::Migration[7.0]
+  def up
+    create_table :aliquot, primary_key: :id, options: 'CHARSET=utf8 COLLATE=utf8_unicode_ci' do |t|
```

Comment:
`id` is the default primary key so we don't have to declare it. However it does make it more verbose so i'm fine to leave it in.